webui/server/server.py

```diff
@@ -151,7 +151,8 @@ def __init__(self, profile_handler, port='', timeout=1):
     self.ser = None
     self.port = port
     self.timeout = timeout
-    self.write_queue = queue.Queue(10)
+    # Sized for `num_panels` threshold updates plus one "v" command
+    self.write_queue = queue.Queue(num_panels + 1)
```

Comment:
In a very brief test I got a "Could not fetch new values. Queue full." warning with 20 panels and write_queue of size 20. A write_queue size 21 worked fine. We could make it larger (num_panels + 10 ?) but I don't have any specific evidence that it would be helpful.

Reply:
Let's just do +10. I think it's okay to have some extra buffer room.